Requirements.md

```diff
@@ -0,0 +1,13 @@
+## Software Requirements Specifications for Spring 2020 "Intro to Software Engineering" "Craft Topography" Project at Wright State Univesity
+### Functional Requirements
+  #### User Story A
```

Comment:
The User Stories are not phrased as User Stories: As a _____, I want _____, so that I can ______.

Reply:
In case it isn't clear, nscale4by8 is an alt for Erik M. Buck. I was logged into the wrong user account when I accepted your invite.